src/pretalx/static/cfp/js/profile.js

```diff
@@ -34,23 +34,6 @@ $(function () {
         }
     });
 
-    $useGravatar.on('change', function () {
-        let gravatarSelected = $useGravatar.prop('checked');
-
-        if (gravatarSelected) {
-            $fileInput.val('');
-            $image.removeClass('d-none');
-            $image.attr('src', "https://www.gravatar.com/avatar/" + $image.data('gravatar') + '?s=512');
-            $resetCheckbox.prop('checked', true);
-        } else if ($image.data('avatar')) {
-            $image.attr('src', $image.data('avatar'));
-            $image.removeClass('d-none');
-            $resetCheckbox.prop('checked', false);
-        } else {
-            $image.addClass('d-none');
-        }
-    });
```

Comment:
Please give me a hint here: Do we want to keep this code, so the submitting user is able to see which avatar will get cached?

Reply:
... but then again we'd need all that CSP stuff again, don't we?

Reply:
Yeah, I'd be in favour of keeping it. Most people won't know/remember/understand what "use gravatar" means, so having a preview is pretty much mandatory.

We'd only need the CSP stuff for these two views (the user's profile page and the profile step in the CfP process, which is still much better than enabling it on all schedule/talk/speaker pages.

Reply:
nah, what i did is only halfway right, let me fix that.

Reply:
Bugs fixed. All views where it's possible to select the gravatar checkbox will now load the image directly from gravatar if the checkbox is checked. The gravatar image is not auto-loaded on page load though, to prevent accidential leaking of ip addresses (especially relevant for orga people if the caching mechanism does not work for any reason).